### PR TITLE
Resolved Issue #39

### PR DESCRIPTION
**Pull request by guranshdeol**
### Issue Resolution #39 
In the latest state, the Css files that were changed and linked with index.html file were present in the root folder rather than /css folder, so I changed the css files in the folder with latest css files and linked the same in the Index.html file.

image

### Button Enhancement:
Additionally for better visuals, changed the Know More Button by adding a border radius to match it with the Click Me button at the bottom, and also added box shadow while hovering for some more aesthetics...

** #Older Look:** 
image
image

**Newer Look:** 
image
image

### CODE REVIEW

**Comment by divyanshudhruv** [comment]
Ok, will see to it in coming days

**Comment by guranshdeol** [comment]
Sure, Thanks😊

Please Don't Forget to label it for Hacktoberfest :)

**Comment by divyanshudhruv** [comment]
I think the style of know more button is ok, no need to change that, but you may fix the issue that is present in the root file and the folder file of css and Javascript . Good luck. But don't change the style of know more button.

**Comment by guranshdeol** [comment]
Done, the changes with the button restored to the old style  and solved the issues.

**Comment by divyanshudhruv** [comment]
Cool, I will merge it after 6oct as I do not currently have the device to merge pr. Hope you didn't mind

**Comment by guranshdeol** [comment]
Yepp No issues...😊

File: index.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Minifolio</title>
     <link rel="shortcut icon" href="Minifolio.png" type="image/x-icon">
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="/css/style.css" />
```
Comment (divyanshudhruv) [inline index.html]:
```suggestion
    <link rel="stylesheet" href="css/style.css" />
```

File: index.html [hunk 2]
```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Minifolio</title>
     <link rel="shortcut icon" href="Minifolio.png" type="image/x-icon">
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="/css/style.css" />
     <script src="https://unpkg.com/ionicons@latest/dist/ionicons.js"></script>
-    <link rel="stylesheet" href="button.css" />
+    <link rel="stylesheet" href="/css/button.css" />
```
Comment (divyanshudhruv) [inline index.html]:
```suggestion
    <link rel="stylesheet" href="css/button.css" />

```

**Comment by divyanshudhruv** [comment]
@guranshdeol  please see the reviews and try to fix those.

**Comment by guranshdeol** [comment]
Which review??

**Comment by divyanshudhruv** [comment]
Wait let me fix that

**Review by divyanshudhruv — APPROVED** [review]
Good job !

**Comment by divyanshudhruv** [comment]
@guranshdeol Congrats

**Pull request merged**